ipalib/constants.py

```diff
@@ -319,3 +319,5 @@
 )
 
 SOFTHSM_DNSSEC_TOKEN_LABEL = u'ipaDNSSEC'
+# certificate verification depth of Apache's mod_ssl (SSLVerifyDepth)
```

Comment:
Please mention that it affects client cert verification only. Perhaps just copy the short explanation from https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslverifydepth ?

``Maximum depth of CA Certificates in Client Certificate verification``